scripts/load_testing/create_account.py

```diff
@@ -52,8 +52,8 @@ def signup(self):
         dom = pyquery.PyQuery(resp.content)
         data = {
             '_method': 'patch',
-            'two_factor_setup_form[phone]': '7035550001',
-            'two_factor_setup_form[otp_delivery_preference]': 'sms',
+            'user_phone_form[phone]': '7035550001',
+            'user_phone_form[otp_delivery_preference]': 'sms',
```

Comment:
I think this might conflict with a PR I just merged....sorry, can you rebase? https://github.com/18F/identity-idp/pull/1572/files